gradle/java.gradle

```diff
@@ -305,4 +307,24 @@ tasks.withType(Test).configureEach {
 tasks.withType(AbstractArchiveTask) {
   preserveFileTimestamps = false
   reproducibleFileOrder = true
-}
\ No newline at end of file
+}
+
+plugins.withId('net.ltgt.errorprone') {
+  dependencies {
+    annotationProcessor group: "com.uber.nullaway", name: "nullaway", version: versions.nullaway
+    errorprone group: "com.google.errorprone", name: "error_prone_core", version: versions.errorprone
+  }
+
+  tasks.withType(JavaCompile) {
+    if (!name.toLowerCase().contains("test")) {
+      options.errorprone {
```

Comment:
My experience with errorprone from Java repo is very negative. I hope it will not became such an annoying asshole as it is over there.

Reply:
You'll see I'm already fairly liberal in disabling checks that are annoying :) There are some checks of questionable value that we can disable when we run into them, on the flip side I have had a few bad bugs caught by it before so do like it in general. Worst case, we'll disableAllChecks to only have nullness checking activated.